OIIInspector/_OIIInspector_calls.py

```diff
@@ -0,0 +1,31 @@
+from utils import setup_arg_parser
+import sys
+
+
+
+GET_INDEX_IMAGE_PACKAGES_LIST_ARGS= {           \
+"--adress" :{                                   \
+    "help": "quay adrres of the index image",   \
```

Comment:
Different typo

Reply:
Also, it doesn't really have to be quay. just "Address of the index image" is fine